src/main/kotlin/com/dropbox/mobile/hypershard/ClassAnnotationValue.kt

```diff
@@ -0,0 +1,10 @@
+package com.dropbox.mobile.hypershard
+
+/**
+ * Annotation name for Kotlin or Java classes
+ *
+ */
+sealed class ClassAnnotationValue {
+    class Present(val annotationName: String) : ClassAnnotationValue()
+    class Empty : ClassAnnotationValue()
```

Comment:
if it is stateless why not `object Empty`?

Reply:
will change